### PR TITLE
DOC: use sphinx docs

### PR DESCRIPTION
**Pull request by llllllllll**
__edit__: this also changes the readme to markdown so pypi renders it correctly and it can be included in the docs

### CODE REVIEW

**Comment by twiecki** [comment]
This is great. Couple of questions:
* Can these be viewed anywhere? 
* Does it upload to gh-pages?
* Do NBs get automatically converted to rst and included in the docs?

**Comment by llllllllll** [comment]
The docs can be built into html, latex, man pages, and a bunch of other formats so I think they should be viewable anywhere. There is an included makefile and also a make.bat for building on windows (though I did not test this).

I am not sure about the uploading of these docs. I know readthedocs accepts sphinx projects as that is how many other projects on readthedocs are documented. What is the process for deploying new documentation?

The notebooks did not automatically get converted; I ran `nbconvert in --to rst`. This could have been automated but there was some manual cleanup needed after the conversion. For example, the levels for each section were too high so I needed to change some of them. There was also some weird output in the terminal output cells that needed to be stripped. The raw notebook is still included though.

File: docs/source/whatsnew/0.8.0.txt [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+~~~~~~~~~~
+
+* New documentation system with a new website at
+  `zipline.io <http://www.zipline.io>`__
+* Major performance enhancements.
+* Dynamic history.
+* New user defined method: ``before_trading_start``.
+* New api function: ``schedule_function``.
+* Pipeline API.
+* Support for trading futures.
```
Comment (richafrank) [inline docs/source/whatsnew/0.8.0.txt]: Other new api functions: `get_environment`, `set_max_leverage`, `set_do_not_order_list`

**Comment by twiecki** [comment]
readthedocs is more trouble than it's worth. I'd just go with gh-pages.

**Comment by llllllllll** [comment]
Okay. Do you know how to update our docs on zipline.io?

**Comment by twiecki** [comment]
Not sure if this is helpful at all: http://docs.hisparc.nl/sapphire/README.html `mkdocs` had a `gh-deploy` command built-in.

File: docs/source/whatsnew/0.8.0.txt [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+Experimental Features
+~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+   Experimental features are subject to change.
+
+* Adds new Pipeline API. The pipeline AAPI is a high-level declarative API for
+  representing trailing window computaions on large datasets (:issue:`630`).
```
Comment (richafrank) [inline docs/source/whatsnew/0.8.0.txt]: typo

File: docs/source/whatsnew/0.8.0.txt [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+* Dynamic history.
+* New user defined method: ``before_trading_start``.
+* New api function: ``schedule_function``.
+* Pipeline API.
+* Support for trading futures.
+
+Enhancements
+~~~~~~~~~~~~
+
+* Account object: Adds an account object to conext to track information about
```
Comment (ssanderson) [inline docs/source/whatsnew/0.8.0.txt]: con**t**ext

File: README.rst [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+
+|Gitter|
+|version status|
+|downloads|
+|build status|
+|Coverage Status|
+|Code quality|
+
+Zipline is a Pythonic algorithmic trading library. The system is
+fundamentally event-driven and a close approximation of how live-trading
```
Comment (llllllllll) [inline README.rst]: should we still say "aprproximation", it _is_ how our live-trading works

File: README.rst [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+systems operate.
+
+Zipline is currently used in production as the backtesting engine
```
Comment (llllllllll) [inline README.rst]: should we mention live trading too?

File: README.rst [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+powering `Quantopian Inc. <https://www.quantopian.com>`__ -- a free,
+community-centered platform that allows development and real-time
+backtesting of trading algorithms in the web browser.
+
+`Join our
+community! <https://groups.google.com/forum/#!forum/zipline>`__
+
+Want to contribute? See our `open
+requests <https://github.com/quantopian/zipline/wiki/Contribution-Requests>`__
```
Comment (llllllllll) [inline README.rst]: audit this, we can do this whenever since it is the wiki

File: README.rst [hunk 7]
```diff
@@ -0,0 +1,181 @@
...
+
+Once set up, you can install Zipline from our Quantopian channel:
+
+::
+
+    conda install -c Quantopian zipline
+
+Currently supported platforms include:
+
+-  Windows 32-bit (can be 64-bit Windows but has to be 32-bit Anaconda)
```
Comment (ssanderson) [inline README.rst]: Is this still true?

File: README.rst [hunk 8]
```diff
@@ -0,0 +1,181 @@
...
+   Python eco-system.
+
+-  Statistic and machine learning libraries like matplotlib, scipy,
+   statsmodels, and sklearn support development, analysis and
+   visualization of state-of-the-art trading systems.
+
+Installation
+============
+
+The easiest way to install Zipline is via ``conda`` which comes as part
```
Comment (ssanderson) [inline README.rst]: `pip` is probably easier until we upgrade the conda packages.

File: docs/source/whatsnew/0.8.0.txt [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+
+Experimental Features
+~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+
+   Experimental features are subject to change.
+
+* Adds new Pipeline API. The pipeline API is a high-level declarative API for
+  representing trailing window computaions on large datasets (:issue:`630`).
```
Comment (abhijeetkalyan) [inline docs/source/whatsnew/0.8.0.txt]: Typo 'computaions' -> 'computations'

File: README.rst (hunk 8 above)
Comment (llllllllll) [inline README.rst]: I have moved pip to suggested method

File: README.rst (hunk 7 above)
Comment (llllllllll) [inline README.rst]: I think I am dropping windows for now. We have not tested this so we cannot make any claims

File: README.rst [hunk 10]
```diff
@@ -0,0 +1,181 @@
...
+install and set up:
+
+::
+
+    pip install numpy   # Pre-install numpy to handle dependency chain quirk
+    pip install zipline
+
+If there are problems installing the dependencies or zipline we
+recommend installing these packages via some other means. For Windows,
+the `Enthought Python
```
Comment (ssanderson) [inline README.rst]: I don't think anyone we know of uses Enthought successfully with Zipline.  We should probably point people on Windows to Anaconda.

File: README.rst [hunk 11]
```diff
@@ -0,0 +1,181 @@
...
+Distribution <http://www.enthought.com/products/epd.php>`__ includes
+most of the necessary dependencies. On OSX, the `Scipy
```
Comment (ssanderson) [inline README.rst]: I'm not sure we need OSX-specific instructions.

File: README.rst [hunk 12]
```diff
@@ -0,0 +1,181 @@
...
+Superpack <http://fonnesbeck.github.com/ScipySuperpack/>`__ works very
+well.
+
+Dependencies
+------------
+
+-  Python (2.7 or 3.3)
```
Comment (ssanderson) [inline README.rst]: We have many more dependencies than these now.  I'm not sure it's worth listing all of them in the readme.

**Comment by llllllllll** [comment]
merged: https://github.com/quantopian/zipline/commit/1955146184fe51f85757ede8813b381898bed049

**Pull request closed without merging**